apisix/cli/ops.lua

```diff
@@ -375,6 +375,10 @@ end
 
 
 local function start(env, ...)
+    if env.is_root_path then
+        util.die("Error: It is forbidden to run APISIX in the /root directory.\n")
```

Comment:
I think you don’t understand the permissions issues under Linux at all

Reply:
the current way is right, but it is hard to read. @Firstsawyou please add some comment here.

![image](https://user-images.githubusercontent.com/6814606/104184835-094b2000-544f-11eb-985d-0a3a352c07f1.png)

Reply:
any news? @Firstsawyou

Reply:
A note has been added here: [https://github.com/apache/apisix/pull/3220/files#diff-c25d1a2aac172147aa01feec3fa29bfe6c4625cda4773719a1cf13e67fd54da2R42 ](https://github.com/apache/apisix/pull/3220/files#diff-c25d1a2aac172147aa01feec3fa29bfe6c4625cda4773719a1cf13e67fd54da2R42)

I will add more comments here later:
https://github.com/apache/apisix/pull/3220/files#diff-228a51571b53709ad043d1c6c14fc8e508c248a84f8014fe548b5d259cedd5ddR378

Reply:
Cause of the bug: After APISIX is started, the `worker process` has the authority of nobody ordinary user. When APISIX is in the /root directory, ordinary users do not have the authority to access the `/root` directory. Therefore, when APISIX is started, the error log `error.log` will show the following error message:

```
init_worker_by_lua error: /root/apisix-2.1/apisix/init.lua:89: module 'resty.worker.events' not found:
        no field package.preload['resty.worker.events']
        no file '/root/apisix-2.1//deps/share/lua/5.1/resty/worker/events.lua'
        no file '/root/apisix-2.1//deps/share/lua/5.1/resty/worker/events/init.lua'
        no file '/root/apisix-2.1/resty/worker/events.lua'
        no file '/root/apisix-2.1/resty/worker/events/init.lua'
```

Reply:
> any news? @Firstsawyou

updated.